ground/src/main/java/com/google/android/ground/ui/offlineareas/OfflineAreasFragment.kt

```diff
@@ -70,7 +71,7 @@ class OfflineAreasFragment : AbstractFragment() {
     super.onViewCreated(view, savedInstanceState)
     lifecycleScope.launch {
       viewModel.navigateToOfflineAreaSelector.collect {
-        findNavController().navigate(OfflineAreasFragmentDirections.showOfflineAreaSelector())
+        findNavController().navigate(R.id.offline_area_selector_fragment)
```

Comment:
Why is this changed required?

Reply:
The earlier one is not working as expected. passing fragment id is more correct approach than passing action id.

Reply:
Using nav actions over view IDs provides type safety, self-documentation of app flows through the nav graph, and make nav intents clearer and more maintainable. Why was `OfflineAreasFragmentDirections.showOfflineAreaSelector()` not working?